sass/components/Dashboard/dropdown.scss

```diff
@@ -0,0 +1,3 @@
+.kubevirt-dropdown__group {
+  margin-right: 0.4em;
```

Comment:
shouldn't this be rather left? because we have the title first

Reply:
ow sorry, probably not - in that case we need to disable the margin for the first or last child (depends on left x right)